src/storage/db_storage.hpp

```diff
@@ -58,13 +58,23 @@ class DbException : public exception {
   string desc_;
 };
 
+struct DbStorage;
+using DB = DbStorage;
+
 struct DbStorage {
-  using BatchPtr = shared_ptr<rocksdb::WriteBatch>;
+  using Slice = rocksdb::Slice;
+  using Batch = rocksdb::WriteBatch;
   using OnEntry = function<bool(Slice const&, Slice const&)>;
 
-  struct Column {
-    string const name;
+  class Column {
+    string const name_;
+
+   public:
     size_t const ordinal;
```

Comment:
why no suffix *_ here?

Reply:
I believe the "_" suffix is only for private fields. Perhaps I got this convention wrong though. To me, a public field with underscore seems quite weird when you access it from outside.

Reply:
I believe you got it wrong. Underscore is used to easily distinguish between class and non-class variables. All class members should have _ suffix. 

https://google.github.io/styleguide/cppguide.html#Variable_Names - "Data members of classes (but not structs) additionally have trailing underscores. For instance: a_local_variable, a_struct_data_member, a_class_data_member_."

Reply:
done